docs/workWithSheet.md

```diff
@@ -0,0 +1,573 @@
+    "requests": [
+      {
+          "addSheet": {
+              "properties": {
+                "title": "Expenses1",
+                "sheetType": "GRID",
+                "gridProperties": {
+                  "rowCount": 50,
+                  "columnCount": 10
+                }
+              }
+          }
+      },
+      {
+          "addSheet": {
+              "properties": {
+                "title": "Expenses2",
+                "sheetType": "GRID",
+                "gridProperties": {
+                  "rowCount": 70,
+                  "columnCount": 10
+                }
+              }
+          }
+      }
+    ],
+      "fields": "spreadsheetId,replies"
+}
+```
+
+
+
+###### Sample Response
+
+```json
+{
+    "spreadsheetId": "1oGxpE3C_2elS4kcCZaB3JqVMiXCYLamC1CXZOgBzy9A",
+    "replies": [
+        {
+            "addSheet": {
+                "properties": {
+                    "sheetId": 372552230,
+                    "title": "Expenses1",
+                    "index": 1,
+                    "sheetType": "GRID",
+                    "gridProperties": {
+                        "rowCount": 50,
+                        "columnCount": 10
+                    }
+                }
+            }
+        },
+        {
+            "addSheet": {
+                "properties": {
+                    "sheetId": 568417391,
+                    "title": "Expenses2",
+                    "index": 2,
+                    "sheetType": "GRID",
+                    "gridProperties": {
+                        "rowCount": 70,
+                        "columnCount": 10
+                    }
+                }
+            }
+        }
+    ]
+}
+```
+
+
+
+###### Related Google Spreadsheet documentation
+
+https://developers.google.com/sheets/reference/rest/v4/spreadsheets/request#AddSheetRequest
+
+
+
+#### Delete Sheet Using BatchRequest
+
+This method allows you to remove sheets from a given spreadsheet using `sheetId`. You can get the `sheetId` using the `getSheetMetaData` method.
+
+
+
+###### deleteSheetBatchRequest
+
+```xml
+<googlespreadsheet.deleteSheetBatchRequest>
+     <spreadsheetId>{$ctx:spreadsheetId}</spreadsheetId>
+     <requests>{$ctx:requests}</requests>
+     <fields>{$ctx:fields}</fields>
+</googlespreadsheet.deleteSheetBatchRequest>
+```
+
+
+
+###### Properties
+
+* spreadsheetId :- Unique value of the spreadsheet
+* requests :- It contains data that is a kind of update to apply to a spreadsheet. To delete multiple sheets within the spread sheet, need to repeat `deleteSheet` property within the `requests` property.
+* fields [Optional] :- Specifying which fields to include in a partial response. For the following request only the `spreadsheetId` will be included in the response.
+
+
+
+###### Sample request
+
+Following is a sample request that can be used for deleteSheetBatchRequest operation. With the following request we can specify Id of the sheet need to be deleted.
```

Comment:
The following sample request calls the deleteSheetBatchRequest operation. Specify the Id of the sheet that needs to be deleted.